animations-baker.js

```diff
@@ -35,6 +35,16 @@ const {CharsetEncoder} = require('three/examples/js/libs/mmdparser.js');
     'Crouched Sneaking Left.fbx',
     'Crouched Sneaking Right.fbx',
   ];
+  const trimClip = (clip, startTime, endTime) => {
```

Comment:
This is ok to ship, but I think we can fix this in the animation itself in the long term.

Maybe add a comment here/open an issue to trim this animation in Blender.

Reply:
Ok, added comment first. https://github.com/webaverse/app/pull/2791/commits/eb1b8da991c6d5c3894dcf9f2f7714ad632ae524
I also want to trim sword animation into three pieces, if this pr can be merged https://github.com/webaverse/app/pull/2804 .
I'll request artist asset afterwards.